### ISSUE TITLE
No suggestion list

### ISSUE THREAD

**Issue by gneil90** [lapism/SearchView#227]
I am trying to add SearchView to activity with following .xml:

```
<?xml version="1.0" encoding="utf-8"?>
<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    tools:context=".Activity.SearchActivity"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <android.support.design.widget.AppBarLayout
        android:id="@+id/appbar"
        android:stateListAnimator="@null"
        android:layout_height="wrap_content"
        android:layout_width="match_parent">

        <android.support.v7.widget.Toolbar
            android:id="@+id/toolbar"
            android:layout_height="@dimen/toolbar_height"
            android:minHeight="@dimen/toolbar_height"
            app:layout_scrollFlags="scroll|enterAlways|snap"

            android:layout_width="match_parent"
            android:layout_weight="1"
            android:layout_marginBottom="15dp"></android.support.v7.widget.Toolbar>

    </android.support.design.widget.AppBarLayout>

    <!--<fragment-->
        <!--android:layout_width="match_parent"-->
        <!--android:layout_height="match_parent"-->
        <!--app:layout_behavior="@string/appbar_scrolling_view_behavior"-->
        <!--android:name="readore.com.readore.fragment.SearchFragment"-->
        <!--android:id="@+id/fragment"-->
        <!--tools:layout="@layout/fragment_search_results" />-->

    <android.support.v7.widget.RecyclerView
        android:id="@+id/recyclerView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:clipToPadding="false"
        android:paddingBottom="4dp"
        android:paddingTop="4dp"
        app:layout_behavior="@string/appbar_scrolling_view_behavior" />

    <com.lapism.searchview.SearchView
        android:id="@+id/searchView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:layout_behavior="com.lapism.searchview.SearchBehavior" />


</android.support.design.widget.CoordinatorLayout>
```

But when I am clicking the SearchView, it does not show suggestion list, however I assign SearchAdapter in OnPostCreate() with three items (like in sample app):

```
        List<SearchItem> suggestionsList = new ArrayList<>();
        suggestionsList.add(new SearchItem("search1"));
        suggestionsList.add(new SearchItem("search2"));
        suggestionsList.add(new SearchItem("search3"));

        SearchAdapter searchAdapter = new SearchAdapter(this, suggestionsList);
        searchAdapter.addOnItemClickListener(new SearchAdapter.OnItemClickListener() {
            @Override
            public void onItemClick(View view, int position) {
                TextView textView = (TextView) view.findViewById(R.id.textView_item_text);
                String query = textView.getText().toString();
                mSearchView.close(false);
            }
        });
        mSearchView.setAdapter(searchAdapter);

```

All I see this (API 21):

![screen shot 2017-01-11 at 4 27 24 pm](https://cloud.githubusercontent.com/assets/3488754/21844958/e0cae232-d81a-11e6-81eb-618ac0ae3a3c.png)

Can anyone help me with setup?

**Comment by AllanWang** [comment]
Did you fix this? I've noticed with the example at least that I need to type something (like "s") before it shows.

**Comment by lapism** [comment]
Fixed for 5.0.
